app/views/layouts/application.html.erb

```diff
@@ -14,6 +14,9 @@
     <meta name="apple-itunes-app" content="app-id=329380089">
     <%= stylesheet_link_tag "tailwind", "data-turbo-track": "reload" %>
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag 'pagy.tailwind', media: 'all' %>
```

Comment:
are these necessary since you included `pagy.tailwind` in the application.tailwind.css file?

Reply:
When I removed it, the pagy navigation bar was displayed without any style. But I just tried removing the import from application.tailwind.css and kept the pagy.tailwind link_tag application.html, and it works now.